src/Components/Form.js

```diff
@@ -0,0 +1,182 @@
+import axios from 'axios';
+import React,{useState} from 'react';
+import DatePicker from 'react-date-picker'
+import { useNavigate } from 'react-router-dom';
+import { FilePicker } from 'evergreen-ui';
+import "./form.css"
+const Form = () => {
+    const [firstname, setFirstname] = useState("");
+    const [lastname, setLastname] = useState("");
+    const [email, setEmail] = useState("");
+    const [mobile, setMobile] = useState("");
+    const [dob, setDob] = useState(new Date());
+    const [course,setCourse]=useState("Maths");
+    const [feedback,setFeedback]=useState("");
+    const [gender,setGender]=useState("");
+    const [agree,setAgree]=useState(false);
+    const [file,setFile]=useState("");
+    const navigate=useNavigate();
+    
+    const handleChangecourse =(e)=>{
```

Comment:
If we had an object instead of individual variables as declared above, we could have one function which could handle the setState for all the variables. Try to think of Re-Usable code wherever possible and ways to shorten the lines of code.

Makes it easier to maintain the code base and looks neat as well.

Reply:
Please keep in mind, you might have to handle the file upload alone separately